scripts/browserBuild.js

```diff
@@ -24,43 +20,64 @@ const babelEs5Options = {
     [
       '@babel/preset-env',
       {
-        // Required for Rollup
-        modules: false,
+        // Required for Webpack
+        modules: 'cjs',
         shippedProposals: true,
         // Target ES5
         targets: 'IE 11',
       },
     ],
     '@babel/preset-flow',
   ],
-  runtimeHelpers: true,
 };
 
 function browserBuild(pkgName, entryPath, destination) {
-  return rollup({
-    input: entryPath,
-    plugins: [
+  rimraf.sync(destination);
+
+  return new Promise((resolve, reject) => {
+    webpack(
+      /* eslint-disable sort-keys */
       {
-        resolveId(id) {
-          return id === 'chalk'
-            ? path.resolve(__dirname, '../packages/expect/build/fakeChalk.js')
-            : undefined;
+        mode: 'development',
+        devtool: 'source-map',
+        entry: entryPath,
+        output: {
+          path: path.dirname(destination),
+          library: camelCase(pkgName),
+          libraryTarget: 'umd',
+          filename: path.basename(destination),
+        },
+        module: {
+          rules: [
+            {
+              test: /\.[jt]sx?$/,
+              loader: 'babel-loader',
+              options: babelEs5Options,
+            },
+          ],
+        },
+        resolve: {
+          alias: {
+            chalk: path.resolve(
+              __dirname,
+              '../packages/expect/build/fakeChalk.js'
+            ),
+          },
+        },
+        node: {
+          fs: 'empty',
```

Comment:
:shrug:

both `jest-message-util` and micromatch (a dependency of `jest-message-util`) uses fs, but work without it